RemoteControl/RemoteControl.json

```diff
@@ -770,6 +921,24 @@
                     "status"
                 ]
             }
+        },
+        "onFirmwareUpdateProgress": {
```

Comment:
Does the app need to know the progression steps of the update? It doesn't matter to me, but if they only care about knowing if it's successful or not, this may not be needed.

Reply:
Gotcha, fine with this